### PR TITLE
adding more margin to footer

### PR DESCRIPTION
**Pull request by georgeteo**

### CODE REVIEW

File: caravel/static/default.css [hunk 1]
```diff
@@ -147,4 +147,13 @@ ul.form-control li {
...
 ul.form-control li label {
     font-weight: normal;
+}
+
+/* Footer */
+div.footer {
+    margin-top: 15px;
+}
+
+footer.footer {
```
Comment (fatlotus) [inline caravel/static/default.css]: Maybe just `footer`? Feels a little redundant.

Comment (georgeteo) [inline caravel/static/default.css]: margin-top isnt a valid css attribute for footers?

Comment (fatlotus) [inline caravel/static/default.css]: wat

**Comment by georgeteo** [comment]
Yeah. You're right. Dunno what I was talking about. I got rid of the div.

**Comment by fatlotus** [comment]
Fix conflicts, then merge away!

**Pull request merged**